src/modules/75-cf/pages/environment-details/EnvironmentDetailsBody.tsx

```diff
@@ -245,18 +247,23 @@ const EnvironmentSDKKeys: React.FC<{ environment: EnvironmentResponseDTO }> = ({
       )}
 
       {emptyData && (
-        <Layout.Vertical width="100%" height="100%" flex={{ align: 'center-center' }} spacing="medium">
-          <Text>{getEnvString('apiKeys.noKeysFound')}</Text>
-          <AddKeyDialog
-            primary
-            environment={environment}
-            onCreate={(newKey: ApiKey, hideCreate) => {
-              setRecents([...recents, newKey])
-              hideCreate()
-              refetch()
-            }}
-          />
-        </Layout.Vertical>
+        <Container flex style={{ flexDirection: 'column' }}>
```

Comment:
This container isn't really doing anything. If you remove the page-specific styling from the `NoData` component, you can add it here. In this case, what you want is to center it in the available space.
```suggestion
        <Container height="100%" flex={{ align: 'center-center' }}>
```

Reply:
Understood, have added the padding in here and adjusted to your suggestions :)